scripts/check-ctrlcode.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import re
+
+def main():
+    args = sys.argv
+    if (len(args) != 3):
+        print("Usage: ./check-ctrlcode.py <dir> <ext>")
+        sys.exit(1)
+
+    dirpath = args[1]
+    ext = args[2]
+
+    fullpath = os.path.abspath(dirpath)
+    if (os.path.isdir(fullpath) is not True):
+        print("Directory not found.")
+        sys.exit(1)
+
+    check_dir(fullpath, ext)
+
+def check_dir(path, ext):
+    for f in os.listdir(path):
+        if(f[0] == "."):
+            continue
+        fullpath = os.path.join(path, f)
+        if(os.path.isfile(fullpath)):
+            exts = os.path.splitext(f)
+            if(exts[1] != ext):
+                continue
+            check_ctrlcode(fullpath)
+        else:
+            check_dir(fullpath, ext)
```

Comment:
similar point. How about changing line 30-34 to
```
if(os.path.isdir(fullpath)):
    check_dir(fullpath, ext)
    continue
exts = os.path.splitext(f)
if(exts[1] != ext):
    continue
check_ctrlcode(fullpath)
```
?

Reply:
I don't think nest is a problem.
I thought about the ease of understanding the conditions.
My code feels easy to understand the if-else condition.
Your code feels easy to understand as a loop condition.
I would like to accept your proposal and fix it.
After fixed, I'll request you to review.

Reply:
This point is not only for nest.
This point makes condition same as the next called function.
In my suggestion, `isdir()` condition calls `check_dir()`.
I think that is easy to understand what is the purpose here.